harvard-library/metadata_server
# LTSK8S-373: logging to stdout for k8s

**LTSK8S-373: logging to stdout for k8s.**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/LTSK8S-373


# What does this Pull Request do?
logs to stdout for k8s compliance

# How should this be tested?
@jaj561 can confirm that this is logging in k8s by going to https://K8S_BASE_URL/manifests/view/drs:400008439

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties: @awoods @jaj561
